src/queries/useReferenceFeesQuery.ts

```diff
@@ -0,0 +1,47 @@
+import { useQuery } from '@tanstack/react-query';
+
+interface Fee {
+  /**
+   * @description medium
+   */
+  m: number;
+  /**
+   * @description high
+   */
+  h: number;
+  /**
+   * @description very high
+   */
+  vh: number;
+}
+
+interface MarketReferenceFee {
+  claim: number;
+  jup: Fee;
+  jup2: Fee;
+  loAndDCA: number;
+  referral: number;
+  perps: Fee;
+  swapFee: number;
+  lastUpdatedAt: number;
+}
+
+export const useReferenceFeesQuery = () => {
+  return useQuery(
+    ['market-reference-fees'],
+    async () => {
+      const data = (await fetch('https://cache.jup.ag/reference-fees')).json() as unknown as MarketReferenceFee;
+      return data;
+    },
+    {
+      keepPreviousData: true,
+      refetchInterval: 60_000,
+      refetchIntervalInBackground: false,
+      retry: false,
+      retryOnMount: false,
+      refetchOnMount: false,
+      refetchOnWindowFocus: false,
+      refetchOnReconnect: false,
```

Comment:
Thinking shall we just disabled all these flag from the root queryprovider level, but not sure is that safe to do since will affect others queries too